CHANGELOG.md

```diff
@@ -3,8 +3,68 @@
 	Placeholder for the next version (at the beginning of the line):
 	## __WORK IN PROGRESS__
 -->
-## 5.0.0 - WORK IN PROGRESS Release Jana
-* Work in Progress
+## 5.0.0 - WORK IN PROGRESS Release Jana (changelog till 13.03.2023)
+**BREAKING CHANGES**
+* Support for Node.js 12 is dropped! Supported are Node.js 14.18.0+, 16.4.0+ and 18.x
+* Backups created with the new js-controller version cannot be restored on hosts with lower js-controller version!
+* Update recommended npm version to 8
+* Deprecate binary states, Adapters will change to use Files instead!
+
+**Features**
+* (foxriver76) Show npm error message on failing adapter installations and update also without debug parameter
+* (foxriver76) Add support for windows for `iob fix`
+* (foxriver76) ?? Allow to block the execution of adapter versions that are know to have bigger issues and add Admin notification
```

Comment:
Why the `??` but yes, basically this is all handled by admin, we are just providing the notification category